group_vars/gxconfig.yml

```diff
@@ -1353,7 +1422,7 @@ base_app_main: &BASE_APP_MAIN
   # configured statsd instance.  The statsd_prefix is useful if you are
   # running multiple Galaxy instances and want to segment statistics
   # between them within the same aggregator.
-  statsd_port: 8125
+  #statsd_port: 8125
```

Comment:
should this line be commented out? Does Galaxy have a default value for this (I am unable to find the default value that Galaxy would set in the case of the absence of this)?

Reply:
https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/schemas/config_schema.yml#L2289